templates/layout.html

```diff
@@ -3,44 +3,46 @@
 {% import 'utility.macros.j2' as utility %}
 {% import 'content.macros.j2' as content %}
 {% import 'layout/macros.j2' as layout %}
+{% import 'contact/macros.j2' as contact %}
 
 {% block layout %}
+
   <header data-region="banner">
-    <nav class="nav">
-      <div class="brand">
-        {% block banner_logo %}
-          <a href="/" rel="home" class="logo-link">
-            {{ utility.icon('logo', alt='OddBird', class='logo-icon') }}
-          </a>
-        {% endblock %}
-      </div>
-
-      {% block banner_nav %}
-        {{ layout.main_nav(config.nav) }}
+    <div class="brand">
+      {% block banner_logo %}
+        <a href="/" rel="home" class="logo-link">
+          {{ utility.icon('logo', alt='OddBird', class='logo-icon') }}
+        </a>
       {% endblock %}
-    </nav>
+    </div>
+
+    {% block banner_nav %}
+      {{ layout.main_nav(config.nav) }}
+    {% endblock %}
 
-    {{ content.divider(small=false) }}
+    {% call content.link_button('/contact/', class='banner-contact') %}
+      {{ utility.icon('mail') }}
+      <span class="banner-contact-text">contact us</span>
+    {% endcall %}
```

Comment:
On hover I get a weird double-line effect on the corners. Is that intentional? I don't hate it, just caught me off-guard.

![screenshot 2018-11-07 12 13 16](https://user-images.githubusercontent.com/552316/48147998-b847ac00-e286-11e8-854a-38480ad9f348.jpg)

Reply:
Not intentional, fixing…